### PR TITLE
US040 - Login System

### PR DESCRIPTION
**Pull request by JakubRS7**

### CODE REVIEW

**Review by lexseaton — DISMISSED** [review]

**Review by RaggedyRach — APPROVED** [review]
Hi Jakub, looks good! Thanks

**Review by RaggedyRach — COMMENTED** [review]
Hi Jakub, i think we need to remove any commented out code.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -23,5 +23,8 @@ jobs:
           node-version: 20.14.0
 
       - name: Run build
-        run: npm ci
+        run: npm install
```
Comment (boydstepkainos) [inline .github/workflows/build.yml]: Should these changes be part of this ticket?

File: .github/workflows/pa11y.yml [hunk 2]
```diff
@@ -30,7 +30,7 @@ jobs:
  
       - name: Install dependencies
 
-        run: npm ci
+        run: npm install
```
Comment (boydstepkainos) [inline .github/workflows/pa11y.yml]: Or this?

**Pull request closed without merging**